src/style.css

```diff
@@ -0,0 +1,124 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+body {
+  font-size: 16px;
+  font-family: "Avenir", sans-serif;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 600;
+}
+
+.search-form--label:after {
+  position: absolute;
+  content: "";
+}
+
+.search-form--label--big:after {
+  top: 25px;
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  position: absolute;
+  content: "";
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-image: url("./images/Search.svg");
+}
+
+.search-form--label--small::after {
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 15px;
+  position: absolute;
+  content: "";
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-image: url("./images/Search.svg");
+}
```

Comment:
You could create a common class for styling label and add modifier for specific style for each label (for example you can style it for a big label and add just **.form__label--small** modifier) like so:

```css
.form__label::after {
  position: absolute;
  content: "";
  background-size: contain;
  background-repeat: no-repeat;
  background-image: url("./images/Search.svg");
  top: 25px;
  left: 26px;
  height: 19px;
  width: 19px;
}

.form__label--small::after {
  top: 15px;
  left: 13px;
  height: 11px;
  width: 11px;
}
```